setup.cfg

```diff
@@ -0,0 +1,33 @@
+[coverage:run]
```

Comment:
most part of the tools for linting/tests at this moment are able to read configuration from `pyproject.toml` file - let's move this config there